src/app/reference/ci-workflows/upload-strategies/page.md

```diff
@@ -226,38 +215,97 @@ const config: PlaywrightTestConfig = {
     },
   ],
 };
+
 export default config;
 ```
 
 {% /tab %}
 {% /tabs %}
 
-## Using GitHub Action
-
-Alternatively, you can upload your replays in a separate step using our [GitHub upload action](https://github.com/replayio/action-upload). To filter which replays to upload, you can use [JSONata filtering functions](https://docs.jsonata.org/higher-order-functions#filter).
-
-```yml {% fileName=".github/workflows/e2e.yml" lineNumbers=true highlight=[19] %}
-name: Replay tests
-on:
-  pull_request:
-  push:
-    branches: [main]
-jobs:
-  cypress-run:
-    runs-on: ubuntu-22.04
-    steps:
-      - name: Checkout
-        uses: actions/checkout@v4
-      - name: Install dependencies
-        run: npm ci
-      - name: Install Replay Chromium
-        run: npx replayio install
-      - name: Run Playwright tests with Replay Browser
-        run: npx playwright test --project replay-chromium --reporter=@replayio/playwright/reporter,line
-      - name: Upload replays
-        if: ${{ always() }}
-        uses: replayio/action-upload@v0.5.1
-        with:
-          api-key: ${{ secrets.REPLAY_API_KEY }}
-          filter: ${{ 'function($v) { $v.metadata.test.result = "failed" }' }}
+## Reducing the number of uploaded recordings
+
+Use the advanced upload options to reduce the number of recordings that are uploaded. When this option is enabled, only one recording will be uploaded for any passing or failing test. For flaky tests, two recordings will be uploaded– the passing test and one of the failed attempts.
+
+```js {% fileName="playwright.config.js" highlight=[7,8,9] %}
+import { replayDevices, replayReporter } from "@replayio/playwright";
+
+const config: PlaywrightTestConfig = {
+  reporter: [
+    replayReporter({
+      apiKey: process.env.REPLAY_API_KEY,
+      upload: {
+          minimizeUploads: true,
+        },
+    }),
+    ["line"],
+  ],
+  projects: [
+    {
+      name: "replay-chromium",
+      use: { ...replayDevices["Replay Chromium"] },
+    },
+  ],
+};
+
+export default config;
+```
+
+{% callout %}
+Note this option is only available for tests recorded with Playwright
+{% /callout %}
+
+## Combining options
+
+You can combine advanced upload options to e.g. only upload a single recording and only for a failing test.
+
+{% tabs labels=["cypress", "playwright"] %}
+{% tab %}
+
+```js {% fileName="cypress.config.js" highlight=[7,8,9,10] %}
+export default defineConfig({
+  e2e: {
+    setupNodeEvents(cyOn, config) {
+      const on = wrapOn(cyOn)
+      replayPlugin(on, config, {
+        apiKey: process.env.REPLAY_API_KEY,
+        upload: {
+          minimizeUploads: true,
```

Comment:
this example uses Cypress but it doesn't support `minimizeUploads`

Reply:
Ah I missed this one. Thanks.